### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AudriusButkevicius**
### Purpose

Adds message about voting on features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ There are a few examples for keeping Syncthing running in the background
 on your system in [the etc directory][3]. There are also several [GUI
 implementations][11] for Windows, Mac and Linux.
 
+## Vote on features/bugs
+
+We'd like to encourage you to [vote][12] on issues that matter to you.
+This helps the team understand what are the biggest pain points for our user, and could potentially influence what is being worked on next.
```
Comment (calmh) [inline README.md]: "users", and maybe s/what/which/ but you're the one in the UK :)

**Comment by AudriusButkevicius** [comment]
I think both what and which works in this context.

**Comment by AudriusButkevicius** [comment]
Amended.

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
@AudriusButkevicius: Merge failed:

```
$ git fetch -f origin refs/pull/3398/head:pr-3398
$ git fetch -f origin master:orig/master
From github.com:syncthing/syncthing
   fbbd510..518f446  master     -> orig/master
$ git reset --hard
HEAD is now at 518f446 cmd/strelaypoolsrv: Fix vet warnings about type inference
$ git checkout master
Already on 'master'
Your branch is up-to-date with 'origin/master'.
$ git reset --hard orig/master
HEAD is now at 518f446 cmd/strelaypoolsrv: Fix vet warnings about type inference
$ git clean -fxd
Skipping repository syncthing/syncthing
Skipping repository syncthing/syncthing-android
Skipping repository syncthing/docs
Skipping repository syncthing/discosrv
Skipping repository syncthing/syncthing-gtk
Skipping repository syncthing/syncthing-inotify
$ git merge --squash --no-commit pr-3398
Updating 518f446..3527fe8
Fast-forward
Squash commit -- not updating HEAD
 README.md | 6 ++++++
 1 file changed, 6 insertions(+)
$ git commit -F -
******************************************************
* Commit message does not match the expected format: *
*                                                    *
*   some/tag: A longer description (fixes #1234)     *
*                                                    *
******************************************************

```

**Comment by AudriusButkevicius** [comment]
@st-review merge

repo: Add message about voting

**Comment by st-review** [comment]
:ok_hand: Merged as f6f0486ff98af18b2c658d401f94d5495c7dacf6. Thanks, @AudriusButkevicius!

**Pull request closed without merging**